rapidtide/workflows/rapidtide_parser.py

```diff
@@ -210,686 +257,933 @@ def _get_parser():
-                        dest='savecorrtimes',
-                        action='store_true',
-                        help='Save a table of lagtimes used. ',
-                        default=False)
-    output.add_argument('--histlen',  # was -h
-                        dest='histlen',
-                        action='store',
-                        type=int,
-                        metavar='HISTLEN',
-                        help=('Change the histogram length to HISTLEN '
-                              '(default is 100). '),
-                        default=100)
-    output.add_argument('--glmsourcefile',
-                        dest='glmsourcefile',
-                        action='store',
-                        type=lambda x: pf.is_valid_file(parser, x),
-                        metavar='FILE',
-                        help=('Regress delayed regressors out of FILE instead '
-                              'of the initial fmri file used to estimate '
-                              'delays. '),
-                        default=None)
-    output.add_argument('--noglm',
-                        dest='doglmfilt',
-                        action='store_false',
-                        help=('Turn off GLM filtering to remove delayed '
-                              'regressor from each voxel (disables output of '
-                              'fitNorm). '),
-                        default=True)
-    output.add_argument('--preservefiltering',
-                        dest='preservefiltering',
-                        action='store_true',
-                        help="Don't reread data prior to performing GLM. ",
-                        default=False)
-    output.add_argument('--saveintermediatemaps',
-                        dest='saveintermediatemaps',
-                        action='store_true',
-                        help='Save lag times, strengths, widths, and mask for each pass. ',
-                        default=False)
-    output.add_argument('--legacyoutput',
-                        dest='bidsoutput',
-                        action='store_false',
-                        help='Use legacy file naming and formats rather than BIDS naming and format conventions for output files. ',
-                        default=True)
+    output = parser.add_argument_group("Output options")
+    output.add_argument(
+        "--nolimitoutput",
+        dest="limitoutput",
+        action="store_false",
+        help=("Save some of the large and rarely used files."),
+        default=True,
+    )
+    output.add_argument(
+        "--savelags",
+        dest="savecorrtimes",
+        action="store_true",
+        help="Save a table of lagtimes used.",
+        default=False,
+    )
+    output.add_argument(
+        "--histlen",  # was -h
+        dest="histlen",
+        action="store",
+        type=int,
+        metavar="HISTLEN",
+        help=("Change the histogram length to HISTLEN (default is 100)."),
+        default=100,
+    )
```

Comment:
I think this conflicts with 

https://github.com/bbfrederick/rapidtide/blob/e613032495e5735b2443b0d0ffa1af2b47574048/rapidtide/workflows/rapidtide_parser.py#L975

Reply:
Yes, line 975 should be deleted.